common/djangoapps/track/views/segmentio.py

```diff
@@ -220,6 +220,10 @@ def track_segmentio_event(request):  # pylint: disable=too-many-statements
 
     context['ip'] = segment_properties.get('context', {}).get('ip', '')
 
+    # For Business Intelligence events: Add label to context
+    if 'label' in segment_properties:
```

Comment:
Ugh, sorry, I meant for this to be part of a review -- not just a "single comment."